### PR TITLE
fix(charts): Update image version to 2.10.1

### PR DESCRIPTION
**Pull request by niladrih**
This PR updates provisioner-localpv image version and Chart version to 2.10.1
Ref: https://github.com/openebs/dynamic-localpv-provisioner/pull/65

### CODE REVIEW

**Comment by niladrih** [comment]
**Note:** Updating the appVersion in Chart.yaml and the release version in values.yaml as well.

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by akhilerm — COMMENTED** [review]
Looks good

**Review by akhilerm — APPROVED** [review]
lgtm

File: deploy/helm/charts/templates/deployment.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ spec:
         - name: OPENEBS_IO_BASE_PATH
           value: "{{ .Values.localpv.basePath }}"
         - name: OPENEBS_IO_HELPER_IMAGE
-          value: "{{ .Values.helperPod.image.registry }}{{ .Values.helperPod.image.repository }}:{{ .Values.localpv.image.tag }}"
+          value: "{{ .Values.helperPod.image.registry }}{{ .Values.helperPod.image.repository }}:{{ .Values.helperPod.image.tag }}"
```
Comment (kmova) [inline deploy/helm/charts/templates/deployment.yaml:77]: 👍

**Review by kmova — APPROVED** [review]

**Pull request merged**